docs/source/cpp/compute.rst

```diff
@@ -627,8 +635,8 @@ The example values are given for default values of ``ndigits`` and ``multiple``.
 +-----------------------+--------------------------------------------------------------+---------------------------+
 
 The following table gives examples of how ``ndigits`` (for the ``round``
-function) and ``multiple`` (for ``round_to_multiple``) influence the operance
-performed, respectively.
+function) and ``multiple`` (for ``round_to_multiple`` and ``round_binary``) 
```

Comment:
Hmm, the second input for `round_binary` is equivalent to `ndigits`, not `multiple`, right?

Reply:
My bad, fixed.